charon/config.py

```diff
@@ -133,6 +157,9 @@ def get_detach_signature_command(self) -> str:
     def is_aws_cf_enable(self) -> bool:
         return self.__aws_cf_enable
 
+    def is_radas_config_enable(self) -> bool:
+        return self.__radas_config_enable
```

Comment:
we can use "radas_config and radas_config.validate()" to decide if it is enabled.